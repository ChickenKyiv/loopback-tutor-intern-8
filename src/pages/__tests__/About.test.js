import React from 'react'
import { shallow } from 'enzyme'
import About from '../Home'

describe('About component', () => {
	it('About renders correctly', () => {
		const wrapper = shallow(<About />);

	    expect(wrapper).toMatchSnapshot();
	})
})