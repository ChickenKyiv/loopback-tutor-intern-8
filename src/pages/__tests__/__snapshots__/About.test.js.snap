// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component About renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
           This is home
        </h1>,
        <div
          className="row"
        >
          <div
            className="col-sm-6 col-md-4"
          >
            <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>
          </div>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " This is home",
        },
        "ref": null,
        "rendered": " This is home",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col-sm-6 col-md-4"
          >
            <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>
          </div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>,
            "className": "col-sm-6 col-md-4",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>,
              "className": "thumbnail",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3>
                    Thumbnail label
                  </h3>,
                  <p>
                    Description about product
                  </p>,
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>,
                ],
                "className": "caption",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Thumbnail label",
                  },
                  "ref": null,
                  "rendered": "Thumbnail label",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Description about product",
                  },
                  "ref": null,
                  "rendered": "Description about product",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Add to cart",
                      "className": "btn btn-primary",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "Add to cart",
                    "type": "button",
                  },
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
             This is home
          </h1>,
          <div
            className="row"
          >
            <div
              className="col-sm-6 col-md-4"
            >
              <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>
            </div>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " This is home",
          },
          "ref": null,
          "rendered": " This is home",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="col-sm-6 col-md-4"
            >
              <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>
            </div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>,
              "className": "col-sm-6 col-md-4",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>,
                "className": "thumbnail",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3>
                      Thumbnail label
                    </h3>,
                    <p>
                      Description about product
                    </p>,
                    <p>
                      <button
                        className="btn btn-primary"
                        onClick={[Function]}
                      >
                        Add to cart
                      </button>
                    </p>,
                  ],
                  "className": "caption",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Thumbnail label",
                    },
                    "ref": null,
                    "rendered": "Thumbnail label",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Description about product",
                    },
                    "ref": null,
                    "rendered": "Description about product",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <button
                        className="btn btn-primary"
                        onClick={[Function]}
                      >
                        Add to cart
                      </button>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Add to cart",
                        "className": "btn btn-primary",
                        "onClick": [Function],
                      },
                      "ref": null,
                      "rendered": "Add to cart",
                      "type": "button",
                    },
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`About renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
           This is home
        </h1>,
        <div
          className="row"
        >
          <div
            className="col-sm-6 col-md-4"
          >
            <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>
          </div>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " This is home",
        },
        "ref": null,
        "rendered": " This is home",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col-sm-6 col-md-4"
          >
            <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>
          </div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="thumbnail"
            >
              <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>
            </div>,
            "className": "col-sm-6 col-md-4",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="caption"
              >
                <h3>
                  Thumbnail label
                </h3>
                <p>
                  Description about product
                </p>
                <p>
                  <button
                    className="btn btn-primary"
                    onClick={[Function]}
                  >
                    Add to cart
                  </button>
                </p>
              </div>,
              "className": "thumbnail",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3>
                    Thumbnail label
                  </h3>,
                  <p>
                    Description about product
                  </p>,
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>,
                ],
                "className": "caption",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Thumbnail label",
                  },
                  "ref": null,
                  "rendered": "Thumbnail label",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Description about product",
                  },
                  "ref": null,
                  "rendered": "Description about product",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Add to cart",
                      "className": "btn btn-primary",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "Add to cart",
                    "type": "button",
                  },
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
             This is home
          </h1>,
          <div
            className="row"
          >
            <div
              className="col-sm-6 col-md-4"
            >
              <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>
            </div>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " This is home",
          },
          "ref": null,
          "rendered": " This is home",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="col-sm-6 col-md-4"
            >
              <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>
            </div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="thumbnail"
              >
                <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>
              </div>,
              "className": "col-sm-6 col-md-4",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="caption"
                >
                  <h3>
                    Thumbnail label
                  </h3>
                  <p>
                    Description about product
                  </p>
                  <p>
                    <button
                      className="btn btn-primary"
                      onClick={[Function]}
                    >
                      Add to cart
                    </button>
                  </p>
                </div>,
                "className": "thumbnail",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3>
                      Thumbnail label
                    </h3>,
                    <p>
                      Description about product
                    </p>,
                    <p>
                      <button
                        className="btn btn-primary"
                        onClick={[Function]}
                      >
                        Add to cart
                      </button>
                    </p>,
                  ],
                  "className": "caption",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Thumbnail label",
                    },
                    "ref": null,
                    "rendered": "Thumbnail label",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Description about product",
                    },
                    "ref": null,
                    "rendered": "Description about product",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <button
                        className="btn btn-primary"
                        onClick={[Function]}
                      >
                        Add to cart
                      </button>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Add to cart",
                        "className": "btn btn-primary",
                        "onClick": [Function],
                      },
                      "ref": null,
                      "rendered": "Add to cart",
                      "type": "button",
                    },
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
